style.css

```diff
@@ -403,4 +412,246 @@ input:focus {
   color: #172b4d;
 }
 
+/* Desktop Version css goes: Using media query */
+
+@media screen and (min-width: 768px) {
+  .mob-nav {
+    display: none;
+  }
+
+  .desktop-nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-left: 120px;
+    padding-right: 120px;
+  }
+
+  .desktop-nav ul {
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+  }
+
+  .desktop-nav p {
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .desktop-nav li {
+    padding: 24px;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .headline-section {
+    background-color: #fff;
+    padding: 50px;
+    background-image: none;
+  }
+
+  .headline-desktop-section {
+    background-image: url(images/desktop-header-bg.svg);
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-size: cover;
+    padding-top: 93px;
+  }
+
+  .headline-desktop-content {
+    padding-left: 44px;
+  }
+
+  .headline-intro {
+    padding-left: 144px;
+    max-width: 920px;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    color: #091e42;
+  }
+
+  .headline-intro-text {
+    max-width: 920px;
+    padding-left: 144px;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+  }
+
+  .headline-social-connect {
+    padding-left: 144px;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .headline-media-links {
+    padding-left: 144px;
+    padding-bottom: 120px;
+  }
+
+  .works-section {
+    display: none;
+  }
+
+  .Desktop-works-section {
+    display: block;
+  }
+
+  .desktop-card-margin {
+    margin: 44px;
+    background: #e5e5e5;
+  }
+
+  .Desktop-works-section .card {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 46px;
+  }
+
+  .Desktop-works-section .card .grid-col-1 {
+    grid-column: 1/2;
+    grid-row: 1/2;
+  }
+
+  .Desktop-works-section .card .grid-col-2 {
+    grid-column: 2/3;
+  }
+
+  .Desktop-works-section .card h2 {
+    margin: 0;
+    padding-top: 18px;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .Desktop-works-section .content-list {
+    margin-top: 0;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .Desktop-works-section .project-content p {
+    max-width: 496px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  main {
+    border-top-right-radius: 50vmax;
+  }
+
+  .about-me-section {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 12px;
+  }
+
```

Comment:
- Please, kindly add some padding to match the design as much as you can.

Reply:
* Required changes ♻️ fixed